### PR TITLE
Remove AssetsHelper from the templating component

### PR DESCRIPTION
**Pull request by alexpts**
> > deprecated since 2.7, will be removed in 3.0. Use the Asset component instead.

### CODE REVIEW

File: components/templating.rst [hunk 1]
```diff
@@ -139,9 +139,8 @@ Helpers
 
 The Templating component can be easily extended via helpers. Helpers are PHP objects that
 provide features useful in a template context. The component has
-2 built-in helpers:
+a built-in helper:
```
Comment (wouterj) [inline components/templating.rst]: "The component has one helper built-in:" ? (I'm not a native, but it sounds a bit better to me)

**Comment by wouterj** [comment]
👍 
status: reviewed

**Comment by wouterj** [comment]
If you have some time left, please also remove the `/components/templating/assetshelper` article (and the references to it).

**Comment by alexpts** [comment]
Ok, I ll pull project and prepare full PR for remove all deps.

**Comment by alexpts** [comment]
PR is ready

**Comment by xabbuh** [comment]
Thank you @alexpts.

**Comment by alexpts** [comment]
I think it need apply to 3.0/3.2 branch also.

**Comment by xabbuh** [comment]
Symfony 3.0 does not receive bug fixes anymore. That's why we also do not update the documentation for it any longer. So I have merged your PR into the `3.1` branch after squashing it and merged it up to master from there. This means that your fix will be part of the documentation for all versions starting with 3.1.

**Pull request closed without merging**